meta-resin-common/recipes-support/resinhup/resinhup.bb

```diff
@@ -9,17 +9,6 @@ inherit allarch
 
 FILES_${PN} = "${bindir}"
 
-RDEPENDS_${PN} = " \
-    bash \
-    busybox \
-    coreutils \
-    docker \
-    e2fsprogs-tune2fs \
-    jq \
-    resin-device-progress \
-    systemd \
-    "
-
 do_install() {
     install -d ${D}${bindir}
     install -m 0755 ${WORKDIR}/run-resinhup.sh ${D}${bindir}
```

Comment:
Is there a need for this entire recipe actually?

Reply:
Well no. I will completely remove it.

Reply:
In this PR or in another?

Reply:
It's part of this PR